### DISCUSSION TITLE
machine-learning/steins-paradox

### DISCUSSION THREAD

**Discussion by joe-antognini** [joe-antognini/joe-antognini.github.io#26] [Comments]
Written on 01/02/2021 23:53:07 

URL: https://joe-antognini.github.io/machine-learning/steins-paradox

**Comment by joe-antognini** [comment]
Comment written by **Bill Vander Lugt** on **10/22/2021 15:28:21**

<p>Thanks for posting.  This was very clear and helpful.</p>

**Comment by joe-antognini** [comment]
Comment written by **T-L-F** on **05/26/2022 15:59:46**

<p>This was a lot of information to take in, it still amazes me how weird statistics can get!~</p>

**Comment by joe-antognini** [comment]
Comment written by **Relun** on **10/19/2022 02:58:29**

<p>In the computation of the Risk of the MLE (initial) estimator for the 1-d Gaussian, where is the 1/sqrt(2*pi) from the pdf of the univariate N(mu, 1) in the integral? I believe the normalizing constant is missing, and the value of the integral shown should be equal to sqrt(2*pi) instead of 1.</p>

**Comment by joe-antognini** [comment]
Comment written by **antognini** on **10/19/2022 16:31:57**

<p>Yes, that's a good catch. I will update the article with the normalization constant.</p>

**Comment by joe-antognini** [comment]
Comment written by **Joseph Gardi** on **01/02/2023 18:06:20**

<p>What is the estimator that beats the james stein estimator?</p>

**Comment by joe-antognini** [comment]
Comment written by **antognini** on **01/02/2023 21:34:49**

<p>I wasn't able to find a specific function of a better estimator.  The sources I read just said that whenever an estimator is discontinuous there exists a continuous estimator that beats it.  But I'm not sure if a functional form for a superior estimator is known.</p>

**Comment by muzhancun** [comment]
This post is amazing and I love the last paragraph!
